test/integration/RemoteMicroservice.test.js

```diff
@@ -237,6 +239,11 @@ describe('The RemoteMicroservice Component', () => {
             expect(this.service.app.models).to.have.property('Author');
         });
 
+        it.skip('does not expose a model on the app if isGlobal is set to false', async function() {
```

Comment:
shouldn't we include this test as it covers the new settings?

Reply:
Then I'd have to start a separate testing suite. If we use an empty configuration (which I need to test as well) we hook in all models. Since Loopback has a global state, I will not be able to test this properly (i could write a unit test which covers this behavior but I'd have to mock parts of the app).